### ISSUE TITLE
Bugs

### ISSUE THREAD

**Issue by joeywen0830** [ahnaf-zamil/light-novel-ocr#2]
Along with DLL missing issue same as the other user, the ocr function sometimes automatically ignores some words, and the final txt file is showing wrong sequences of sentences compared to the original image text. Would you be kind enough to explain what kind of image should we use to receive the most accurate outcome? Thanks.

**Comment by ahnaf-zamil** [comment]
Hello there, thanks for making the issue!

Are you using the version from the Releases page? Shouldn't say DLL missing if you used that. Although, Windows can be a b*itch sometimes. What specific DLL is it missing? It should say that in a popup prompt.

Also, please send the input images and I'll try to test them out myself. I personally didn't train the JP_VERT detection model that is used for the text detection, it was trained by the guys who made Tesseract. But completely ignoring some words is very strange, since it has never happened.

The sequencing/ordering of sentences may be an issue caused by my ordering algorithm, I'll have to take a look at that.

Thanks for making the issue though :)

**Comment by joeywen0830** [comment]
> Hello there, thanks for making the issue!
> 
> Are you using the version from the Releases page? Shouldn't say DLL missing if you used that. Although, Windows can be a b*itch sometimes. What specific DLL is it missing? It should say that in a popup prompt.
> 
> Also, please send the input images and I'll try to test them out myself. I personally didn't train the JP_VERT detection model that is used for the text detection, it was trained by the guys who made Tesseract. But completely ignoring some words is very strange, since it has never happened.
> 
> The sequencing/ordering of sentences may be an issue caused by my ordering algorithm, I'll have to take a look at that.
> 
> Thanks for making the issue though :)

Yeah I'm using the ones from the releases. The DLL issue is just like what the other user's image shows. About the missing word issue, I'm not sure why, sometimes it happens
